Source/Core/TileAvailability.js

```diff
@@ -1,17 +1,17 @@
 /*global define*/
 define([
+    './Check',
```

Comment:
I think you included this file by accident (did you branch from master? Or branch from #5552 by accident)? Anyway, no problem, just remove the changes in this file.